README.rst

```diff
@@ -40,6 +40,14 @@ and remove these three lines from ``.bashrc``:
     eval "$(pyenv init -)"
     eval "$(pyenv virtualenv-init -)"
 
+and restart your shell so the path changes take effect.
+
+.. code:: bash
+
+    $ exec $SHELL
+
+You can now begin using pyenv.
```

Comment:
You added those lines in the **uninstall** section. 

But I generally agree that the instructions could use some TLC - maybe also a better separation between install and uninstall instructions. Keep at it and we'll get there :)